sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCSuite.scala

```diff
@@ -638,7 +638,7 @@ class JDBCSuite extends QueryTest
     assert(rows(0).getAs[Array[Byte]](0).sameElements(testBytes))
     assert(rows(0).getString(1).equals("Sensitive"))
     assert(rows(0).getString(2).equals("Insensitive"))
-    assert(rows(0).getString(3).equals("Twenty-byte CHAR"))
+    assert(rows(0).getString(3).equals("Twenty-byte CHAR    "))
```

Comment:
what happens here?

Reply:
The data type is `CHAR(20)`. ref: https://github.com/apache/spark/blob/0819210f7a88a77efb20e8c786ba01b30f2b9b05/sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCSuite.scala#L149
It seems the newer H2 append the other space.

Reply:
Yes, new result is correct. It's nice to have this update.

https://github.com/apache/spark/blob/4caface4b635bc339da370ac644855f47d46b01f/sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCSuite.scala#L154